schemas/string_interpolation/definitions.json

```diff
@@ -152,5 +166,42 @@
     },
     "additionalProperties": false,
     "required": ["source", "identifier"]
+  },
+  "text_plural": {
+    "description": "A mapping of a count to plural strings",
+    "type": "object",
+    "properties": {
+      "forms": {
+        "description": "Plural forms of the text. Uses the CLDR plural categories. One or more of these properties will be present dependent on the language.",
+        "type": "object",
+        "properties": {
+          "zero": {
+            "type": "string"
+          },
+          "one": {
+            "type": "string"
+          },
+          "two": {
+            "type": "string"
+          },
+          "few": {
+            "type": "string"
+          },
+          "many": {
+            "type": "string"
+          },
+          "other": {
+            "type": "string"
+          },
```

Comment:
No, because that would allow an object with a `placeholders` within each plural form i.e.

```json
{
    "summary_text": {
        "placeholders": [
            {
                "placeholder": "number_of_people",
                "transforms": []
            }
        ],
        "text_plural": {
            "forms": {
                "one": "Yes, {number_of_people} person lives here",
                "other": {
					"text": "Yes, {number_of_people} people live here",
					"placeholders": [
						{}
					]
				}
            },
            "count": {
                "source": "list",
                "identifier": "household"
            }
        }
    }
```

The intent is that the `text_plural` element is resolved to a string equivalent of a `text` element, which is then combined with the resolved placeholders. If there were more placeholders than the number in the sentence then it would look something like:

```json
{
    "summary_text": {
        "placeholders": [
            {
                "placeholder": "number_of_people",
                "transforms": [
                {
                    "arguments": {
                        "number": {
                            "source": "list",
                            "identifier": "household"
                        }
                    },
                    "transform": "number_to_words"
                }
                ]
			{
				"placeholder": "address",
                "value": {
					"source": "metadata",
					"identifier": "display_address"
				}
            }
        ],
        "text_plural": {
            "forms": {
                "one": "Yes, {number_of_people} person live at {address}",
                "other": "Yes, {number_of_people} people live at {address}"
            },
            "count": {
                "source": "list",
                "identifier": "household"
            }
        }
    }
```

I'll add this as an example to the document.

Reply:
Ah ay, I confused myself because of the parenthesis.